src/domain/event/eventHero/__tests__/EventHero.test.tsx

```diff
@@ -47,8 +47,12 @@ afterAll(() => {
   clear();
 });
 
+const renderComponent = (props?: Partial<EventHeroProps>) => {
```

Comment:
props seems to be redundant (it only could have `event` and `eventType` which are already set)

Reply:
also you could use `eventType={EventType.EVENT}`

Reply:
> props seems to be redundant (it only could have `event` and `eventType` which are already set)

I don't they are redundant, the idea is that you can override the default props that are set in the function for the component



> also you could use `eventType={EventType.EVENT}`

true, changed/fixed

Reply:
<img width="757" alt="Screenshot 2020-12-31 at 11 35 57" src="https://user-images.githubusercontent.com/15219142/103404472-5d572a00-4b5c-11eb-97ed-27cf5d9876d1.png">

no params are passed in any test atm but if we would add more tests and override the props the typing is useful